assignments/a6.md

```diff
@@ -51,7 +56,33 @@ It is suggested to use a folder and multiple files, just like this CSC491 repo,
 
 # Rubric
  
-**TODO**
+| Area | Description| Weight |
+| --- | --- | --- |
+| Applications | All applications present in the architecture diagram have a repository and an MVP is implemented. | 20.0 |
+| Test coverage | Tests should cover at least 65% of your application, upload to the DCSIL Team App, and upload to Code Climate. | 20.0 |
+| Code Quality | All repositories have at least a "B" Grade on Code Climate. | 10.0 |
```

Comment:
When marking is it simply a threshold to get a 10.0 on this section?

Reply:
yea. I thought about other ways to mark it, but we don't really have the bandwidth to review _all_ of their code